### PR TITLE
Feat seed database cronjob

### PR DESCRIPTION
**Pull request by liewshanhong**
To run script: 
python manage.py runscript -v2 seed_database

To start cron in docker:
- go to docker CLI
- enter 'service cron start'

To check status of cron in docker:
- go to docker CLI
- enter 'service cron status'

To run cron:
- add job to cron 
'python manage.py crontab add'

### CODE REVIEW

File: skills_DS/scripts/seed_database.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
...int-Sauveur', 'Oromocto', 'Kitimat', 'La Malbaie', 'Stony Plain', 'Mont-Tremblant', 'Binbrook', 'Whitecourt', 'Lavaltrie', 'Ladysmith', 'Pont-Rouge', 'Castlegar', 'Otterburn Park', 'Dauphin', 'Bois-des-Filion', 'Winkler', 'Concord', 'Bracebridge', 'Notre-Dame-des-Prairies', 'Dryden', 'Taber', 'Morinville', 'Silver Berry', 'Saint-Felicien', 'Saint-Hippolyte', 'Bridgewater', 'Goderich', 'Wild Rose', 'Renfrew', 'Edson', 'Fort Frances', 'Chandler', 'Kirkland Lake', 'Slave Lake', 'Happy Valley-Goose Bay', 'Wasaga Beach', 'Chibougamau', 'Revelstoke', 'Aylmer', 'Pont Rouge', 'Camlachie', 'Banff', 'Yarmouth', 'Carignan', 'Campbellton', 'Beaumont', 'Innisfil', 'Delson', 'Sydney Mines', 'Baie-Saint-Paul', 'Hanover', 'La Sarre', 'Asbestos', 'Merritt', 'Bluewater', 'Coaticook', 'Hampstead', 'Olds', 'Greenwood', 'Gibsons', 'Carbonear', 'Saint-Zotique', 'Kincardine', 'Plessisville', 'Brownsburg-Chatham', 'Ponoka', 'Mont-Joli', 'Rocky Mountain House', 'Kimberley', 'Perth', 'Parry Sound', 'Pointe-Calumet', 'Morden', 'Coaldale', 'Summerland', 'Stephenville', 'Amqui', 'Beauceville', 'Saint-Joseph-du-Lac', 'Sooke', 'Iqaluit', 'Drayton Valley', 'Farnham', 'The Pas', 'Bromont', 'Saint-Felix-de-Valois']
```
Comment (SanaBob) [inline skills_DS/scripts/seed_database.py:12]: this seems like a lot of data for a constant. I would rename this to LOCATION or CONSTANTS_LOCATION but this is also fine.

File: skills_DS/scripts/seed_database.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
...Social worker', 'Software developer', 'Soil scientist', 'Solicitor', 'Speech and language therapist', 'Sports coach', 'Sports development officer', 'Sports therapist', 'Statistician', 'Stockbroker', 'Structural engineer', 'Systems analyst', 'Systems developer', 'Tax inspector', 'Teacher (nursery/early years)', 'Teacher (primary)', 'Teacher (secondary)', 'Teacher (special educational needs)', 'Teaching/classroom assistant', 'Technical author', 'Technical sales engineer', 'TEFL/TESL teacher', 'Television production assistant', 'Test automation developer', 'Tour operator', 'Tourism officer', 'Tourist information manager', 'Town and country planner', 'Toxicologist', 'Trade union research officer', 'Trader', 'Trading standards officer', 'Training and development officer', 'Translator', 'Transportation planner', 'Travel agent', 'TV/film/theatre set designer', 'UX designer', 'Validation engineer', 'Veterinary nurse', 'Veterinary surgeon', 'Video game designer', 'Video game developer', 'Volunteer work organiser', 'Warehouse manager', 'Waste disposal officer', 'Water conservation officer', 'Water engineer', 'Web designer', 'Web developer', 'Welfare rights adviser', 'Writer', 'Youth worker']
```
Comment (SanaBob) [inline skills_DS/scripts/seed_database.py:39]: same here

**Review by SanaBob — APPROVED** [review]

**Comment by SanaBob** [comment]
Not sure if this PR is done yet, but you also need to update Readme, if there's update on running the docker

**Comment by liewshanhong** [comment]
waiting for one more review

**Review by brianzhouzc — APPROVED** [review]

**Pull request merged**